src/app/state/rest/index.ts

```diff
@@ -240,4 +240,41 @@ export const useReasons = (options?: Options) => {
     isProtected: true,
     isMocked: true
   })
-}
\ No newline at end of file
+}
+
+export const useOpinions = (options?: Options) => {
+  const handleError = useErrorHandler()
+  return useApiRequest<MockComponents.Schemas.Opinion>({
+    ...options,
+    url: "opinion",
```

Comment:
I don't see an `opinion` key exported from `src/__mocked__/data.ts`? Also the variable name is plural: `useOpinions`?

Reply:
Never mind. See it's only used for `execPost`.